MAUI/Cartesian-Charts/Axis/Overview.md

```diff
@@ -16,6 +16,7 @@ Vertical(Y) axis always uses numerical scale. Horizontal(X) axis supports the fo
 * Category
 * Numeric
 * Date time
+* Logarithmic
```

Comment:
Change to logarithm

Reply:
@Saravanan-Madhesh , Requested change is now committed.